tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -142,7 +142,7 @@ export class Ide {
         await this.driverHelper.waitAndClick(By.xpath(yesButtonLocator));
     }
 
-    async waitWorkspaceAndIde(timeout: number = TimeoutConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT) {
```

Comment:
IMHO it would be better not to change TS_SELENIUM_LOAD_PAGE_TIMEOUT constant name in here, because it has quite a relevant semantic related to IDE.
Alternatively we can increase TS_SELENIUM_LOAD_PAGE_TIMEOUT timeout number in TimeoutConstants file.

Reply:
I see. It's just not always start of workspace, but just load of the page when workspace has already started before and is now just reloading.

Reply:
But due to provided screen `TS_SELENIUM_START_WORKSPACE_TIMEOUT` is suitable in here as well.